libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -661,7 +661,7 @@ void AP_ICEngine::update_idle_governor(int8_t &min_throttle)
     // Double Check to make sure engine is really running
     if (!ap_rpm->get_rpm(rpm_instance-1, rpmv) || rpmv < 1) {
         // Reset idle point to the default value when the engine is stopped
-        idle_governor_integrator = min_throttle;
+        idle_governor_integrator = start_percent.get();
```

Comment:
Would it be better to set `idle_point_initialized = false` here? Maybe we should do the same for the return on 655?